relay-spans/src/v2_to_v1.rs

```diff
@@ -38,9 +38,10 @@ pub fn span_v2_to_span_v1(span_v2: SpanV2) -> SpanV1 {
         status,
         is_remote,
         other: _other,
-    } = span_v2;
+    } = span_v2.clone();
```

Comment:
You're cloning the entire V2 span here so you can later call `derive_op_for_v2_span` on it. There are a few ways around this.
1. Call the function before deconstructing the V2 span. This has the minor flaw that you may not need the result, but calling that function unnecessarily is still preferable to cloning unnecessarily.
2. Don't deconstruct the V2 span at all.